spec/views/user_show_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.feature 'User Show', type: :feature do
+  let(:user) { User.create(name: 'Tom', photo: 'https://www.kasandbox.org/programming-images/avatars/leaf-blue.png', bio: 'He is a good programmar') }
+  let!(:post1) { Post.create(author: user, title: 'first post', text: 'first text') }
+  let!(:post2) { Post.create(author: user, title: 'second post', text: 'second text') }
+  let!(:post3) { Post.create(author: user, title: 'third post', text: '3 text') }
+  let!(:post4) { Post.create(author: user, title: '4 post', text: '4 text') }
+
+  scenario 'visiting the user Show page' do
+    visit user_path(user)
+
+    expect(page).to have_content('Tom')
+    expect(page).to have_css("img[alt='Tom']", count: 1)
+  end
+
+  scenario 'visiting the user show page, you see the number of posts the user has written..' do
+    visit user_path(user)
+
+    expect(page).to have_content('4 posts')
+  end
+
+  scenario 'visiting the user show page, you see the 3 most recent post and bio of the user has written..' do
+    visit user_path(user)
+    expect(page).to have_content('He is a good programmar')
+    expect(page).not_to have_content('fist text')
+  end
+
+  scenario 'visiting the user show page, you see the 3 most recent post and bio of the user has written..' do
+    visit user_path(user)
+    expect(page).to have_content('He is a good programmar')
+    expect(page).not_to have_content('fist text')
+  end
```

Comment:
- [ ] Please you are repeating this test scenario :pray:

Reply:
- [ ] [OPTIONAL] Could you remove this test case scenario as it is currently repeated?